### DISCUSSION TITLE
Aesthetic scoring doesn't seem to work at clip-front ui.

### DISCUSSION THREAD

**Discussion by SharDi0** [rom1504/clip-retrieval#243] [Q&A] [answered]
I am using clip front ui in web and I'm trying to make use of aesthetic scoring option but it doesn't change results at all. I've read it worked before and showed results I need, but not anymore. Is there anything I could do to make it work?

**Comment by rom1504** [comment]
Works only with L/14 ; you can switch to it

We could make it work for H i guess @nousr

**Reply by SharDi0** [reply]
still no changes after switching to L-14. I also must note that this issue appears for other people too.

**Comment by nousr** [comment]
@rom1504 it seems @SharDi0 is correct -- i wasn't able to get aesthetic weighting to work with the L/14 model ... maybe i accidentally disabled it in the config? checking now

regarding creating aesthetic embeddings for h/14 -- i guess its possible, we an discuss in DM about what that would entail

**Comment by nousr** [comment, marked as answer]
@SharDi0 it should work now with the L/14 and B/32 indices

**Reply by SharDi0** [reply]
thank you very much for fixing it, while we're on this topic about aesthetic weighting, is there anywhere where I could learn what exactly does score and weight do and how it works works with images?

**Comment by rom1504** [comment]
I computed prediction by Laion aesthetic V1 in buckets from 0 to 10
Then i averaged the laion2B L/14 embeddings for each bucket
That gave a 0 embedding, a 1 one, etc

What the UI does is take your query embedding (eg image or text embedding)
and do a weighted average with it
The score is the bucket. The weight is the weight given to the aesthetic
embedding in this average
